gardener/main.py

```diff
@@ -1,13 +1,20 @@
 #!/usr/bin/env python3
-from settings import accounts
+from settings import accounts, mode, switchoff
 from behavior import behavior, Todolist
 from utils import bcolors, g
 from lwapi import lwapi
+import os
 
 #################################################################
 # Main program
 #################################################################
-
+switchoff = False
```

Comment:
switchoff = false by default to avoid issues where the user forgot to do something with this setting